### ISSUE TITLE
Not working only got a white screen and no error message.

### ISSUE THREAD

**Issue by jmolivas** [Maslosoft/oauth2-twitter#1]
I add this library to my project using composer:

```
composer require maslosoft/oauth2-twitter:1.0.x-dev
```

Then setup my `config.yaml` file like:

```
knpu_oauth2_client:
    clients:
        twitter:
            type: generic
            provider_class: League\OAuth2\Client\Provider\Twitter

            client_id: APP_CONSUMER_KEY
            client_secret: APP_CONSUMER_SECRET

            redirect_route: connect_twitter_check
            redirect_params: {}
```

Add a TwitterController class and register two controller actions.

```
    /**
     * @Route("/connect/twitter")
     */
    public function connectAction()
    {
       ... code here ...
    }

    /**
     * @Route("/connect/twitter/check", name="connect_twitter_check")
     */
    public function connectCheckAction(Request $request)
    {
       ... code here ...
    }
```

When calling route `connect/twitter`

Got the URL redirected to `https://api.twitter.com/oauth2/token?state=` and a white screen no error message shown.

**Comment by pmaselkowski** [comment]
Sadly twitter does not allow authenticating via oauth2. oauth2 is only used for applications authorization. To allow authenticating users with twitter you need to use oauth (version 1) and obtain permission from twitter to receive email (if you need email).

**TLDR;** This project is on hold until twitter fully supports oauth2

**Comment by witchgamez** [comment]
Hey there !

Firstable, thank you so much for your bundle, really easy to use, i save a lot of time ;)

I think Twitter have OAuth2 now (https://developer.twitter.com/en/docs/authentication/oauth-2-0). Have you planned to implement this connector ?

Thx !
